### PR TITLE
Pager fixes

### PR DESCRIPTION
**Pull request by jenswi-linaro**
Picks up the patch from https://github.com/OP-TEE/optee_os/issues/1153 and also fixes another pager issue.

### CODE REVIEW

File: core/arch/arm/mm/tee_pager.c [hunk 1]
```diff
@@ -615,6 +615,11 @@ static size_t area_va2idx(struct tee_pager_area *area, vaddr_t va)
 	return (va - (area->base & ~CORE_MMU_PGDIR_MASK)) >> SMALL_PAGE_SHIFT;
 }
 
+static vaddr_t __unused area_idx2va(struct tee_pager_area *area, size_t idx)
```
Comment (jforissier) [inline core/arch/arm/mm/tee_pager.c]: `__maybe_unused`

**Comment by jenswi-linaro** [comment]
Update

**Comment by jforissier** [comment]
`Reviewed-by: Jerome Forissier <jerome.forissier@linaro.org>`

**Comment by jenswi-linaro** [comment]
Squashed, tag applied

**Pull request merged**